nemo/collections/llm/api.py

```diff
@@ -326,7 +326,8 @@ def deploy(
     model_type: str = "llama",
     triton_model_name: str = "triton_model",
     triton_model_version: Optional[int] = 1,
-    triton_port: int = 8000,
+    triton_http_port: int = 8000,
+    triton_grpc_port: int = 8001,
     triton_http_address: str = "0.0.0.0",
```

Comment:
Nit: Can we change `triton_port` to `triton_http_port` in the docstring as well ? 
Also can we add `triton_grpc_port` in the docstrings ?

Reply:
@athitten  Thanks! Just changed the docstring accordingly.